src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -179,7 +199,7 @@
 
 (def required-ca-files
   "The set of SSL related files that are required on the CA."
-  #{:cacert :cacrl :cakey :cert-inventory :serial})
+  #{:cacert :cacrl :cakey :cert-inventory :serial :infra-node-serials-path :infra-crl-path})
```

Comment:
We talked about splitting this between the infra-crl related files and not and then in the caller conditionally combining if `enable-infra-crl` is true